### DISCUSSION TITLE
Community Connect Event ?

### DISCUSSION THREAD

**Discussion by parthlawate** [Sunbird-Ed/Community#248] [General Discussion]
Hello All, 

Over the last few week's i've had the opportunity to engage deeply with a lot of folks in the community across organisations & ended up discussing Sunbird Building blocks , what they were building with it, what challenges they had & how the community could collaborate better to avoid any duplication of work. 

Starting this thread to see if there is some interest to have an Online 'Community Connect' event around Sunbird ED where we can discuss how we can align various downstream work tracks so we can avoid duplication & improve collaboration. Here is a format in which the event could run. 

Showcase & Feedback 
A Community implementor /organisation will show case what they are building using Sunbird BB's & their technical approach to extending them if any . 

Community members can then brainstorm and give feedback on the 'How' of the implementation, call out if any of them have aligned activities which could be potential collaboration opportunities & even propose alternatives. 

Thoughts ? 

@rahul101001000 @pramodkvarma @surendrasinghs

**Comment by surendrasinghs** [comment]
@parthlawate this sounds like a good start. I believe the format can also include some highlights on the usage of the solution and insights from it. 

@Krishnaj20 @sharathkashyap @Sunbird-Ed/sunbirded_workinggroup

**Reply by rahul101001000** [reply]
Fantastic idea @parthlawate ! This could follow the Sunbird RC series format and once special interest groups form based on use case areas, this could evolve to Beckn like calls where specs and sample apps get built. Sunbird Ed can help so many organizations!
